client/modules/User/pages/SignupView.jsx

```diff
@@ -26,19 +27,19 @@ class SignupView extends React.Component {
         <Nav layout="dashboard" />
         <main className="form-container">
           <Helmet>
-            <title>p5.js Web Editor | Signup</title>
+            <title>{this.props.t('SignupView.Title')}</title>
           </Helmet>
           <div className="form-container__content">
-            <h2 className="form-container__title">Sign Up</h2>
+            <h2 className="form-container__title">{this.props.t('SignupView.Description')}</h2>
             <SignupForm {...this.props} />
-            <h2 className="form-container__divider">Or</h2>
+            <h2 className="form-container__divider">{this.props.t('SignupView.Or')}</h2>
             <div className="form-container__stack">
               <SocialAuthButton service={SocialAuthButton.services.github} />
               <SocialAuthButton service={SocialAuthButton.services.google} />
             </div>
             <p className="form__navigation-options">
-              Already have an account?&nbsp;
-              <Link className="form__login-button" to="/login">Log In</Link>
+              {this.props.t('SignupView.AlreadyHave')}
```

Comment:
"AlreadyHave" doesn't really describe the purpose of this label... "HaveAccount" is more accurate?

I think we don't need to change this instance, but maybe something to think about for other things that are being translated.